internal/kafka/command_partition_reassignment_list.go

```diff
@@ -91,13 +101,31 @@ func (c *partitionCommand) reassignmentList(cmd *cobra.Command, args []string) e
 
 	list := output.NewList(cmd)
 	for _, reassignment := range reassignments.Data {
-		list.Add(&getReassignmentOut{
-			ClusterId:        reassignment.ClusterId,
-			TopicName:        reassignment.TopicName,
-			PartitionId:      reassignment.PartitionId,
-			AddingReplicas:   reassignment.AddingReplicas,
-			RemovingReplicas: reassignment.RemovingReplicas,
-		})
+		if output.GetFormat(cmd) == output.Human {
+			list.Add(&getReassignmentOutHuman{
+				ClusterId:        reassignment.ClusterId,
+				TopicName:        reassignment.TopicName,
+				PartitionId:      reassignment.PartitionId,
+				AddingReplicas:   join(reassignment.AddingReplicas),
```

Comment:
how does it print other []string? can we just do the join in the `output` package